city_scrapers/spiders/chi_ssa_20.py

```diff
@@ -0,0 +1,105 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa20Spider(CityScrapersSpider):
+    name = "chi_ssa_20"
+    agency = "Chicago Special Service Area #20 South Western Avenue"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+    location = {
+        "name": "Beverly Bank & Trust,",
+        "address": "10258 s. Western ave.",
+    }
+
+    def parse(self, response):
+
+        base = response.xpath(
+            "//*[self::p or self::strong or self::h3]/text()"
+        ).getall()
+
+        base = [re.sub(r"\s+", " ", item).lower() for item in base]
+
+        for index, line in enumerate(base):
```

Comment:
Is there a reason for iterating through this separately? A more standard way of doing this would be what you're doing below where we check a conditional and `continue` if it isn't met.

Reply:
There was no special reason, just stuck with what worked initially :neutral_face: .
I moved this down to line 40 and it's handled within the main loop `for item in base:`